websites/S/SDVX Index/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "189690228292845568",
+		"name": "angelolz"
+	},
+	"service": "SDVX Index",
+	"description": {
+		"en": "A chart viewer for Sound Voltex, providing top-down overviews of charts for players to study and practice."
+	},
+	"url": "sdvxindex.com",
+	"matches": [
+		"*://www.sdvxindex.com/*",
+		"*://sdvxindex.com/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/0bkkYu0.png",
+	"thumbnail": "https://i.imgur.com/mWrqTjr.png",
```

Comment:
Currently, looks like Imgur is failing with this image, it's a 404. If it comes back, everything else looks fine. However, it may be worth it to upload the image to a different image host.

Reply:
Weird. I just tried to load both these images now and they work absolutely fine for me.

Reply:
Yea thumbnail seems to be failing for me, logo seems fine.
Please try uploading to a different host, or I don't know if you were logged into Imgur when uploading? Because I think they auto-delete real fast if you aren't logged in.

Reply:
I was logged in when I uploaded this. It's still currently in my own Imgur account.

Either way, I've tried using a different [service](https://imgbb.com/). Let me know if that worked for you!